deploy-tool/cert/kubernetes_storage.go

```diff
@@ -48,6 +51,7 @@ func cleanKey(key string) string {
 
 var dataKey = "value"
 
+// Store stores a key and value inside Kubernetes secret storage
```

Comment:
I like the doccomment, and appreciate the commitment to conventional style. If you're doing CI stuff in other PRs, I recommend putting a linter in, so that we never have to argue about style again.